src/Server.jl

```diff
@@ -12,7 +12,8 @@ using JSON
 using MbedTLS
 
 import Pkg: TOML
```

Comment:
This should be changed to just `import Pkg` no? I see that Server.jl uses `Pkg.Types.read_project` in a few places.

Reply:
Well it needs `TOML`, and there was already a `using Pkg` above

Reply:
Ah, you're right. Somehow I missed that.